core-plugins/src/main/java/co/cask/hydrator/plugin/streaming/Window.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.plugin.streaming;
+
+import co.cask.cdap.api.annotation.Description;
+import co.cask.cdap.api.annotation.Name;
+import co.cask.cdap.api.annotation.Plugin;
+import co.cask.cdap.etl.api.streaming.Windower;
+
+/**
+ * Window plugin.
+ */
+@Plugin(type = Windower.PLUGIN_TYPE)
+@Name("Window")
+@Description("Windows a pipeline.")
+public class Window extends Windower {
```

Comment:
just curious - why is there a Windower api? What are some other types of windows? (perhaps some not based upon time, but is that even supported by the API?)